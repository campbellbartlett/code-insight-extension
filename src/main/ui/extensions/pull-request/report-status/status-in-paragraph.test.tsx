import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';
import StatusInParagraph from './status-in-paragraph';

describe('status in paragraph renders correctly', () => {
    it('should match snapshot', () => {
        const statusInParagraph = shallow(<StatusInParagraph text="Test Status" color="red" />);

        expect(statusInParagraph).not.toBeNull();
        expect(shallowToJson(statusInParagraph)).toMatchSnapshot();
    });
});
