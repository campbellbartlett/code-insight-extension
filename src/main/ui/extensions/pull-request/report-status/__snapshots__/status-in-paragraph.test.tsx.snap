// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`status in paragraph renders correctly should match snapshot 1`] = `
<div
  style={
    Object {
      "width": "4em",
    }
  }
>
  <WithAnalyticsEvents(StatusInternal)
    color="red"
    text="Test Status"
  />
</div>
`;
